client/src/test/scala/org/http4s/client/middleware/FollowRedirectSpec.scala

```diff
@@ -42,7 +42,7 @@ class FollowRedirectSpec extends Http4sSpec with Tables {
       def doIt(method: Method, status: Status, body: String, pure: Boolean, response: Throwable \/ RedirectResponse) = {
         val u = uri("http://localhost") / status.code.toString
         val req = method match {
-          case _: Method with Method.PermitsBody if body.nonEmpty =>
+          case _: Method.PermitsBody if body.nonEmpty =>
```

Comment:
Scala 2.11 complained of a fruitless type until `PermitsBody` extended `Method`. Even a self-type wasn't enough.  What's up with this?